### ISSUE TITLE
Expand on Traverse queries through OrientJS

### ISSUE THREAD

**Issue by OjasM** [orientechnologies/orientdb#6728]
## OrientDB Version, operating system, or hardware.
- v2.2.8
## Operating System
- Linux

## Expected behavior and actual behavior
Query  --> Traverse in('Investment') from #24:0 
When fired on console or Studio, returns all expected records completely.
When fired through query() function of OrientJS, only first record is returned completely, but only the record ids of all the other records is returned.

I also tried --> Select expand(result) from (traverse in('Investment') as result from #24:0) as suggested [here](https://groups.google.com/forum/#!topic/orient-database/iyb0792nOvU)
But that doesn't work.

## Steps to reproduce the problem
Fire the sample query through query() function of OrientJS.

**Comment by maggiolo00** [comment]
@OjasM 

do you have a script to reproduce this issue?

**Comment by OjasM** [comment]
@maggiolo00 
The one attached reproduces the error.
[test.txt](https://github.com/orientechnologies/orientdb/files/484761/test.txt)

**Comment by maggiolo00** [comment]
@OjasM 
do you have also some script sql with data so i can test on a fresh db?

**Comment by OjasM** [comment]
@maggiolo00 

You can try for the "GratefulDeadConcerts" db.

``` javascript 
var OrientDB = require('/usr/local/lib/node_modules/orientjs');

conn_params = {
                hostname: 'localhost',
                port: 2424,
                user: 'root',
                password: 'password'
            };

var server = OrientDB(conn_params);
            
var db = server.use({
name:     'GratefulDeadConcerts',
user:     'admin',
password: 'admin'
});

db
.query("traverse in('written_by') from (Select from V where name = 'Garcia')")
.then(function(data){
	console.log(data);
});
```

**Comment by OjasM** [comment]
The records are completely returned ( as expected ) on adding the "select from ( <sub-query> )" as suggested.

**Comment by maggiolo00** [comment]
Ok 
let me check the problem without select

**Comment by maggiolo00** [comment]
@OjasM 

Seems that it's related to OrientDB not to OrientJS

**Comment by maggiolo00** [comment]
hi @OjasM 

I've fixed it in ODB. It will be available in 2.2.11

**Comment by OjasM** [comment]
@maggiolo00 
Thanks
